sre-recipes/sre_recipes.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+####################################################################
+## This CLI handles the deploying broken services and rolling
+## broken services back. 
+####################################################################
+
+import os
+import subprocess
+import click
+
+dir_abs_path = os.path.dirname(os.path.abspath(__file__))
+manifest_abs_path = dir_abs_path + '/../kubernetes-manifests/'
+
+# Returns the absolute path of the working service's kubernetes manifest
+# and the broken service's kubernetes manifest
+def get_paths(service):
+    working_service_abs_path = manifest_abs_path + service + ".yaml"
+    broken_service_abs_path = manifest_abs_path + service + "srerecipes.yaml"
+    return working_service_abs_path, broken_service_abs_path
+
+
+# Runs the given command and returns any output and error
+def run_command(command):
+    process = subprocess.Popen(command.split(), stdout=subprocess.PIPE)
+    output, error = process.communicate()
+    return output, error
+
+
+# Rolls back the working version of the given service and deploys the
+# broken version of the given service
+def break_func(service):
+    working_service_abs_path, broken_service_abs_path = get_paths(service)
+
+    print("Rolling back working " + service + " service...")
+    delete_command = 'kubectl delete -f ' + working_service_abs_path
+    output, error = run_command(delete_command)
```

Comment:
What if it failed to delete?

Reply:
Good question! From what I've seen, the delete fails if the service that we want to remove isn't there. In this case, I think it's fine to do nothing because the point is to remove the service anyway. I'm not quite sure about the case where the delete fails because kubernetes actually failed to delete. I'll some investigating on this after I put in changes for the CLI restructure.

Reply:
I think this will no longer be an issue because we aren't going to use different images to switch between working and broken versions anymore.